src/python/turicreate/toolkits/object_detector/object_detector.py

```diff
@@ -182,12 +182,16 @@ def create(dataset, annotations=None, feature=None, model='darknet-yolo',
         >>> data['image_pred'] = turicreate.object_detector.util.draw_bounding_boxes(data['image'], data['predictions'])
     """
     _raise_error_if_not_sframe(dataset, "dataset")
-    from ._mx_detector import YOLOLoss as _YOLOLoss
-    from ._model import tiny_darknet as _tiny_darknet
-    from ._sframe_loader import SFrameDetectionIter as _SFrameDetectionIter
-    from ._manual_scheduler import ManualScheduler as _ManualScheduler
-    import mxnet as _mx
-    from .._mxnet import _mxnet_utils
+
+    if not USE_CPP:
+        # Use MxNet
+        from ._model import tiny_darknet as _tiny_darknet
+        import mxnet as _mx
+        from .._mxnet import _mxnet_utils
+        from ._mx_detector import YOLOLoss as _YOLOLoss
+        from ._sframe_loader import SFrameDetectionIter as _SFrameDetectionIter
+        from ._manual_scheduler import ManualScheduler as _ManualScheduler
+
```

Comment:
Could we just move these down until they are needed? It seems weird to import them conditionally here